src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -295,4 +295,14 @@ std::array<double, 3> spin_vector(double spin_magnitude,
                                   const Scalar<DataVector>& ricci_scalar,
                                   const Scalar<DataVector>& spin_function,
                                   const YlmSpherepack& ylm) noexcept;
+
+/*!
+ * \ingroup SurfacesGroup
+ * \brief Irreducible mass of a 2D `Strahlkorper`.
+ *
+ * \details See Eqs. (15.38)
+ * \cite hartle2003gravity . This function computes the irreducible mass from the area of a
```

Comment:
`\cite` does not have to be at the beginning of the line. You can add it to the previous line.
You don't need the white space after hartle2003gravity.
Please adhere to the 80 characters limit. (There should have been a check to prevent this from being commited. How did you get around that?)

Reply:
I suspect editing directly in a web browser is how you can get around this check. 

@jennifersanchez please put `\cite hartle2003gravity` on the previous line and then run 
`git-clang-format-5.0 -f src/ApparentHorizons` on this file, which will automatically enforce the 80 character limit.

Reply:
Ah yea, I don't think there's any way for us to enforce the limit through GitHub's web interface (other than travis complaining :) )